tests/librFSM/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 
 
 # LoadrFSM
-ADD_RTF_CPPTEST(NAME LoadrFSMTest
-                SRCS loadrFSM.cpp
-                PARAM "${CMAKE_SOURCE_DIR}/tests/fsm/simple_fsm.lua")
-
+include_directories(../../librFSM/include)
+ADD_RobotTestingFramework_CPPTEST(NAME LoadrFSMTest
+                                  SRCS loadrFSM.cpp
+                                  PARAM "${CMAKE_SOURCE_DIR}/tests/fsm/simple_fsm.lua")
```

Comment:
How is this working if rFSM is not linked?

Reply:
I asked to myself the same thing :thinking: 
It used to work, let's see if it still work :sweat_smile:

Reply:
I suspect it is using some header-only part of the library.

Reply:
It didn't work actually, it gave linking problem at runtime